test/Orleans.Serialization.UnitTests/BuiltInCodecTests.cs

```diff
@@ -36,7 +36,7 @@ public enum MyEnum : short
     public class EnumTests : FieldCodecTester<MyEnum, IFieldCodec<MyEnum>>
     {
         protected override IFieldCodec<MyEnum> CreateCodec() => ServiceProvider.GetRequiredService<ICodecProvider>().GetCodec<MyEnum>();
-        protected override MyEnum CreateValue() => (MyEnum)(new Random(Guid.NewGuid().GetHashCode()).Next((int)MyEnum.None, (int)MyEnum.Two));
+        protected override MyEnum CreateValue() => (MyEnum)(Random.Shared.Next((int)MyEnum.None, (int)MyEnum.Two));
```

Comment:
If a test fails the test-run isn't reproducable, as the seed is unknown.
So I would either
* use a constant seed
* create a random seed and use it, but also log that seed

Thus test-failures can be replayed easily (and hence easier fixed).

Reply:
I'll log the seed on class instantiation - it will be a substantial change.